### DISCUSSION TITLE
Filament Change Button

### DISCUSSION THREAD

**Discussion by fiveorders** [bigtreetech/BIGTREETECH-TouchScreenFirmware#1863] [Ideas]
Dear All is it possible to put a Filament Change Button basically will be a shortcut to M600

I know there are custom commands, but looking for a stand-alone filament change button possibility

**Comment by kisslorand** [comment]
Why? What's wrong with custom commands?
You have already the (Un)Load menu that i just perfect for the filament change.

**Reply by fiveorders** [reply]
I am having strange behaviours with unload command. first, extruder motor turns very fast for a second and then waits 3-4 seconds and starts to unload. Can you send your marling settings for the filament load-unload process?

**Reply by fiveorders** [reply]
Also tried two times, and these lines of error occurred when resuming.

![IMG_0610](https://user-images.githubusercontent.com/13808989/115539147-87ac2f00-a294-11eb-9af6-04999719b223.JPG)

**Reply by kisslorand** [reply]
For the moment I can't, I'm at work where I do not have my Marlin, but as I recall I left everything at default speed. Those rapid short movements are not any kind of error, it is made for a better unload of the filament.
